app/src/main/java/org/simple/clinic/ClinicApp.kt

```diff
@@ -86,6 +91,11 @@ abstract class ClinicApp : Application(), CameraXConfig.Provider {
   }
 
   private fun setupApplicationPerformanceMonitoring() {
+    val samplingRate = remoteConfig
+        .double("datadog_sample_rate", 0.0)
```

Comment:
Currently we have this turned off?

Reply:
It's turned off by default, I have enabled it on Remote Config consoles.